packages/components/tests/integration/components/hds/table/th-sort-test.js

```diff
@@ -14,31 +14,44 @@ module('Integration | Component | hds/table/th-sort', function (hooks) {
 
   test('it renders', async function (assert) {
     await render(hbs`<Hds::Table::ThSort
-  @sortBy='artist'
-  @sortOrder='asc'
   @setSortBy={{this.setSortBy}}
 >artist</Hds::Table::ThSort>`);
 
     assert.dom(this.element).hasText('artist');
   });
+  test('it has the scope attribute, and it is set to column', async function (assert) {
+    await render(hbs`<Hds::Table::ThSort
+  @setSortBy={{this.setSortBy}}
+>artist</Hds::Table::ThSort>`);
+
+    assert.dom('[data-test-table-th-sort]').hasAttribute('scope', 'col');
+  });
 
   test('it should render the component with a CSS class that matches the component name', async function (assert) {
     await render(hbs`<Hds::Table::ThSort
-  @sortBy='artist'
-  @sortOrder='asc'
   @setSortBy={{this.setSortBy}}
 >artist</Hds::Table::ThSort>`);
 
     assert.dom('[data-test-table-th-sort]').hasClass('hds-table__th-sort');
   });
+
   test('if @sortOrder is not defined, the swap-vertical icon should be displayed', async function (assert) {
     await render(hbs`<Hds::Table::ThSort
   @sortBy='artist'
   @setSortBy={{this.setSortBy}}
 >artist</Hds::Table::ThSort>`);
 
-    assert
-      .dom(this.element.querySelector('.flight-icon.flight-icon-swap-vertical'))
-      .exists();
+    assert.dom('[data-test-icon="swap-vertical"]').exists();
+  });
+
+  test('if `@sortOrder` is set to asc, the arrow-up icon should be displayed', async function (assert) {
+    await render(hbs`<Hds::Table::ThSort
+  @sortBy='artist'
+  @sortKey='artist'
+  @sortOrder='asc'
+  @setSortBy={{this.setSortBy}}
+>artist</Hds::Table::ThSort>`);
```

Comment:
[nitpick] I am not sure why this strange formatting. have you tried to use prettier on the page?

Reply:
yes, prettier is the one that formatted it this way. 🤷